Lib/test/test_pathlib.py

```diff
@@ -30,6 +30,20 @@ def test_is_notimplemented(self):
         self.assertTrue(isinstance(pathlib.UnsupportedOperation(), NotImplementedError))
 
 
+try:
+    import pwd
+    all_users = [u.pw_uid for u in pwd.getpwall()]
+except (ImportError, AttributeError):
+    all_users = []
+
+
+try:
+    import grp
+    all_groups = [g.gr_gid for g in grp.getgrall()]
+except (ImportError, AttributeError):
+    all_groups = []
+
+
```

Comment:
Could these be moved to a helper method, or inline in the tests that use em? They're not used unless the test suite is being run as root, which is a fairly rare configuration. You can call `self.skipTest()` to skip from the test method body.

Reply:
Makes sense. Done 🙂